post_office/management/commands/load_email_template.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from ...models import EmailTemplate
+from ...logutils import setup_loghandlers
+
+logger = setup_loghandlers()
+
+
+def _get_template(template_name):
+    """Find the template and return the raw template file content"""
+
+    template = loader.get_template(template_name)
```

Comment:
Does this line fail with a clear error message if `loader` can't find the specified template? In my experience, this could be a source of frustration if the user doesn't know where he's supposed to put the files.